.github/workflows/ci_rails.yml

```diff
@@ -0,0 +1,77 @@
+# GitHub Actions Continuous Integration of MO Rails Code
+# Runs the tests and publishes the results to Coveralls
+name: Continuous Integration
+
+on: 
+  push:
+    # branches: master
+    branches: 
+      - "*" # Run on pushes on all branches
+  pull_request:
+    branches: master
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04 # focal, includes MySQL 5.8
+    steps:
+    # check-out repo under $GITHUB_WORKSPACE, so that workflow can access it.
+    # https://github.com/actions/checkout
+    - name: Checkout code
+      uses: actions/checkout@v2  
+
+    # https://github.com/actions/setup-ruby
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs bundle install, caches gems
+
+    - name: Install additional tools
+      run: sudo apt-get install exiftool
+
+    # MySQL is installed but does not run by default
+    # https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#mysql
+    - name: Start mySQL
+      run: sudo systemctl start mysql.service
+      
+    - name: Create and configure db
+      run: |
+        mysql -u root -proot < db/initialize.sql
+        cp db/vagrant/database.yml config
```

Comment:
From `script/travis_setup`.  Questions:
- **Is it correct?** (It works, but I don't understand what it does.)
- I got rid of a bunch of conditionals from `script/travis_setup`. **Should I instead retain them?**

Reply:
This looks right to me.  It creates and sets up the test database.

re: conditionals -- Is there any chance it will retain the test environment from one build to another?  If not, if it starts from scratch with every single build, then yeah, there's no point in the conditionals.